addon/services/ember-freestyle.js

```diff
@@ -22,6 +21,23 @@ export default Ember.Service.extend({
 
   notFocused: computed.not('focus'),
 
+  ensureHljs() {
+    if (!hljsPromise) {
+      hljsPromise = new Promise((resolve) => {
+        return Ember.$.getScript('https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.11.0/highlight.min.js').done((script) => {
```

Comment:
Or maybe this will just work as is. We could add a `catch` that logs the fact that syntax highlighting is not available.